nfs-22-04/marketplace-image.json

```diff
@@ -0,0 +1,60 @@
+{
+  "variables": {
+    "token": "{{env `DIGITALOCEAN_TOKEN`}}",
+    "image_name": "Bikram-NFS-server-{{timestamp}}"
```

Comment:
I guess `Bikram` can be removed from image name, no?

Reply:
Yes